packages/ckeditor5-utils/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Just an example to make VS Code working. It'll be refined along with the tooling that is using this.